docs/reference/composites.rst

```diff
@@ -0,0 +1,271 @@
+.. _included_composites:
+
+==========
+Composites
+==========
+
+`dwave~system` provides :std:doc:`dimod composites <dimod:reference/samplers>`
+for using the D~Wave system.
```

Comment:
What's with the tildas instead of hyphens?

Reply:
oops, a replace-all gone awry